display.py

```diff
@@ -145,11 +158,32 @@ def runGame(self, numbots):
 					DrawButton.disable()
 					my_player.stay(gamestate)
 					gamestate.waitforplayer = False
+				if ResolveButton.check(event):
+					# end game when all players done
+					if all([not player.playing for player in gamestate.players]):
+						self.showPostgame()
+					gamestate.waitforplayer = False
```

Comment:
If anything set this to True. But it shouldn't matter once `endGame()` has been called.